src/server.js

```diff
@@ -11,7 +11,9 @@ function server ({ pathPrefix, redisUrl, isRedisCluster, defaultExpiration }, fa
       const options = parseOptions(request.query)
       return client.session(request.params.ns, request.params.sid, {
         flags: options.flags,
-        duration: defaultExpiration
+        duration: isNaN(options.duration)
+          ? defaultExpiration
+          : options.duration
```

Comment:
I don't think that the client is going to behave well if it receives `duration <= 0`.
I see two options here:
1. Validate here at the server that `duration >= 1`
2. Or change the client to accept `0` as duration

Anyway, we need to at least validate here that the duration is a non-negative integer.

Reply:
How about https://github.com/escaletech/tog-session-server/pull/1/commits/87aeb596da411d494b86fa103e42644fba45ded5?